mbta.rb

```diff
@@ -1,5 +1,76 @@
 
 # your solution here
-def stops_between_stations
 
-end
\ No newline at end of file
+#===Methodes for travling in the same line====#
+def stops_in_one_line(line, start_station, end_station)
+    lines = {
+        red: ['South Station', 'Park Street', 'Kendall', 'Central', 'Harvard', 'Porter', 'Davis', 'Alewife'],
+        green: ['Government Center', 'Park Street', 'Boylston', 'Arlington', 'Copley', 'Hynes', 'Kenmore'],
+        orange: ['North Station', 'Haymarket', 'Park Street', 'State', 'Downtown Crossing', 'Chinatown', 'Back Bay', 'Forest Hills']
+    }
+    
+        
+    #getting the index
+    start_index = lines[:"#{line}"].index(start_station)
+    end_index = lines[:"#{line}"].index(end_station)
```

Comment:
```ruby
    start_index = lines[line.to_sym].index(start_station)
```
[`.to_sym`](https://apidock.com/ruby/String/to_sym) is the simplest way to turn your string into a symbol